services/code.service.js

```diff
@@ -119,61 +118,65 @@ const _prepareErrorMessage = (outputLog, language, command) => {
 }
 
 const _executePrompt = async (
+    count,
     langConfig,
     prompt,
     response,
-    maxPoints = 10,
-    systemPrompt,
+    points = 10,
```

Comment:
No particular reason, I just though of keeping the same naming convention everywhere because in all the repos of ai evaluation flow we know it as points and then suddenly calling it maxPoints may get confusing is what I feel? but if needed will change it